inbm/dispatcher-agent/tests/unit/sota/test_os_factory.py

```diff
@@ -31,7 +31,7 @@ def test_get_factory_windows(self) -> None:
     def test_raise_error_unsupported_OsFactory(self) -> None:
         factory = SotaOsFactory(self.mock_disp_broker)
         with self.assertRaises(ValueError):
-                factory.get_os("MacOS")
+            factory.get_os("MacOS")
```

Comment:
<h3>$\color{#15F4EE}Miscellaneous: CodeTesting$</h3>

Ensure that the updated unit tests reflect the changes made in the corresponding modules and that they cover all new code paths introduced.

Reply:
out of scope--this is from reformatting the code base